db/migrate/20180403094945_increase_node_properties_limit.rb

```diff
@@ -0,0 +1,5 @@
+class IncreaseNodePropertiesLimit < ActiveRecord::Migration[5.1]
+  def change
+    change_column :nodes, :properties, :text, limit: 4.gigabytes - 1
```

Comment:
this number is absurd, wouldn't it be a waste of resources?

Reply:
I'm assuming it's just the limit and in an ideal scenario this amount is never reached. This limit is also the same for the `versions` table `object` column.

Reply:
so we are just using that big number because this forces rails to use a `longtext` column in the database?

We are using a different number in the versions table?  https://github.com/dradis/dradis-ce/blob/master/db/migrate/20151119124120_create_versions.rb#L7

I have also seen someone doing it like this: https://gist.github.com/ozgun/7672578
Does this work?

Reply:
@dormi In using the mysql adapter, the value in the versions migration forces the column to be of `longtext` type (see the schema after migration).

Updated the PR with the `:longtext` option.